test/e2e/secret_test.go

```diff
@@ -336,6 +336,16 @@ var _ = Describe("Podman secret", func() {
 		Expect(session.OutputToStringArray()).To(HaveLen(1))
 	})
 
+	It("podman secret rm --ignore", func() {
+		remove := podmanTest.Podman([]string{"secret", "rm", "non-existent-secret"})
+		remove.WaitWithDefaultTimeout()
+		Expect(remove).Should(Not(Exit(0)))
```

Comment:
A good test checks error messages, not just status code. Because `should-not-exit-0` would happily pass if you had a syntax error in your test command, or any other unrelated error. Recommendation: add something like this
```go
       Expect(remove.ErrorToString()).To(Equal("Error: no secret with name or id \"non-existent-secret\": no such secret"))
```